internal/pkg/cli/dialog/inputs_select.go

```diff
@@ -22,29 +21,24 @@ import (
 	"github.com/keboola/keboola-as-code/internal/pkg/utils/strhelper"
 )
 
-type templateInputsDialog struct {
+// inputsSelectDialog to select which config/row fields will be replaced by user input.
+type inputsSelectDialog struct {
 	prompt       prompt.Prompt
 	options      *options.Options
 	branch       *model.Branch
 	configs      []*model.ConfigWithRows
+	inputs       inputsMap
```

Comment:
`inputsMap` je spolocna pre oba dialogy, a preto je to teraz uz zavislost, dostane sa do sktrukury externe, cez `newInputsSelectDialog`.